### PR TITLE
LPS-180070 | Test Automation | Master

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-180070](https://issues.liferay.com/browse/LPS-180070) ObjectActions#CanAddUserAfterCreatingCommerceProductEntry

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by victorhcunha** [comment]
ci:test:object

**Comment by victorhcunha** [comment]
Tests using the same part of the macro passed ci:

- CanActivateOrDeactivateAction
- CanCreateActionAddAnObjectEntryUsingOldValueWithOnAfterUpdateTrigger
- CanCreateActionAddObjectEntryWithOnOrderStatusUpdate
- CanCreateActionAddObjectEntryWithOnPaymentStatusUpdate
- CanCreateActionUpdateAnObjectEntryUsingOldValueWithOnAfterUpdateTrigger
- CanCreateActionWithExpressionBuilder
- CanCreateEntryWithActions
- CanEditExpression
- CanEnableAndDisableCondition
- CanTriggerActionAfterDisablingExpression
- CanTriggerActionWithExpressionByAddingEntry
- CanTriggerStandaloneActionWithPermission
- CanTriggerActionWithExpressionByUpdatingEntry
- CheckStandaloneActionPermissionOfUnpublishedObject
- CanCreateActionWithCustomFieldInSystemObjects
- CanDeleteNotification

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 1]
```diff
@@ -1273,6 +1265,23 @@ definition {
...
+			Click(
+				key_rowNumber = ${rowNumber},
+				locator1 = "ObjectAction#INPUT_METHOD");
+
+			Type(
+				key_rowNumber = ${rowNumber},
+				locator1 = "ObjectAction#INPUT_VALUE",
+				value1 = ${value});
+		}
+	}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:1283]: This is a neat little solution

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 2]
```diff
@@ -106,6 +106,64 @@ definition {
...
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Product");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.assertObjectActions(
+				actionActive = "Yes",
+				actionLabel = "Custom Action",
+				status = "Success");
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:163]: We can omit these steps. The scope of the test is to assert the user is added. The assertion you're making here should be tested under an Actions smoke test.

**Review by timpak — CHANGES_REQUESTED** [review]
@victorhcunha can you take a look at my comment? Pretty interesting solution you found for multiple input values btw.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase (hunk 2 above)
Comment (victorhcunha) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase]: Right! Omitted steps

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro (hunk 1 above)
Comment (victorhcunha) [reply modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:1282]: Thank you!

**Review by timpak — APPROVED** [review]

**Pull request closed without merging**